vars/Windows.yml

```diff
@@ -0,0 +1,4 @@
+sensu_client_config: "windows_client.json.j2"
+sensu_win_rabbitmq_config: "windows-sensu-rabbitmq.json.j2"
+sensu_msi_download_path: "http://repositories.sensuapp.org/msi/2012r2/sensu-1.2.1-2-x86.msi"
+sensu_msi_product_id: "{FF98EE93-B344-4548-ADD2-E2F1BD217A84}"
```

Comment:
Since I don't really use Windows, could you describe where you pulled this value from? Is it straight from the MSI package?

Reply:
Yes, this comes from the MSI. You can find it in the registry after the package is installed. So this UID changes with each package. Windows requires we provide this as a hard identifier for the installed package.